packages/conf-openblas/conf-openblas.0.2.0/opam

```diff
@@ -6,17 +6,14 @@ bug-reports: "https://github.com/ocaml/opam-repository/issues"
 dev-repo: "https://github.com/ocaml/opam-repository.git"
 license: "BSD"
 build: [
-  ["sh" "-exc" "cc $CFLAGS test.c -lopenblas"] {os = "alpine"}
-  ["sh" "-exc" "cc $CFLAGS test.c -lopenblas"] {os = "debian"}
```

Comment:
`debian` has been removed. This shouldn't be the case I suppose

Reply:
This case should still be covered by the following line (`{os != "fedora" ...}`) I think?

Reply:
Yes, you are right. Sorry.